docs/README.md

```diff
@@ -1,5 +1,11 @@
 # Documentation
 
+## What if?
```

Comment:
Feel this would be better under `Users`. Could maybe be grouped with a line break at first?